notebooks/02_foundations__connection_riemannian_metric.ipynb

```diff
@@ -0,0 +1,2266 @@
+{
```

Comment:
<p>NIT: euclidean -&gt; Euclidean</p><p> </p><p>NIT: when we say that a&nbsp;vector field is parallel <strong>along a curve&nbsp;</strong>on a manifold</p><p> </p><p>First remember that a vector field&nbsp;X&nbsp;is comprised of a set of vectors X_p \in T_pM where p can be any point on the manifold M. &gt; &gt;&gt; Because:</p><ul><li>p represents a point on the manifold</li><li>t is usually a time parameter, typically a real number in [a, b]</li></ul><p> </p><p>Then, consider that a vector field X along a curve gamma, is comprised of vectors X_(gamma(t)) \in T_gamma(t)M at the tangent space of each point gamma(t) of the curve gamma, when the parameter t varies in [a, b]. (i.e. consider p = gamma(t) in the previous expressionve). We can write X(t) \in T_gamma(t)M for short.</p><p> </p><p>Now, a parallel vector field X along a curve gamma is a vector field along gamma that additionally verifies: \nabla_\dot gamma (t) X(t) = 0</p><p> </p><p>"has a "zero connection" with the curve's tangent vector"</p><p>&gt;&gt; Unfortunately, I am not sure that we can really say this :/ I have never heard it, I think "connection" comes from the fact that we connect the computations of different tangent spaces.</p><p> </p><p>""Note that this means that in differential geometry, we can only define the term "parallel" when we define it&nbsp;along a curve</p><p>along a curve"" -&gt;&gt; Yesss :D Maybe this sentence could come slightly earlier in the notebook (where I have put remarks)?</p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>agree with most of what you said, but wrote something slightly different: </p><p> </p><p>Now, consider a vector field $\textbf{X}$ that lies along a curve $\gamma(t)$. This means that each one of the vectors in $\textbf{X}$ stems from a point along $\gamma(t)$. This vector field is a $\textbf{parallel vector field}$ if it satisfies:</p><p> </p><p>$$\nabla_{\dot{\gamma}(t)} x(t)=0$$.</p><p> </p><p>In other words: if we were to move each point in the vector field $\textbf{X}$ infinitesimally along the curve $\gamma$ in the direction of $\gamma$'s tangent vector $\dot{\gamma}$, the vector field $\textbf{X}$ would not change. Note that this means that in differential geometry, we can only define the term "parallel" when we define it $\textit{along a curve}$.</p>